src/olympia/search/tests/test_search_ranking.py

```diff
@@ -1063,3 +1063,10 @@ def test_scenario_promoted(self):
                 ['Stripy Dog 4', 584],  # not promoted
             ),
         )
+
+    def test_scenario_minimum_should_match_trigrams(self):
```

Comment:
This test scenario is hard to follow. Is it possible to mock the config for 66 and 67 and have an assert for both to verify modifying that value specifically modifies the expected outcome? This test is probably something that will randomly fail in the future due to some other partially related change and it will end up in a PR similar to Tabby Cat.

Reply:
These tests care about scoring so the way we have set them up is to have an entire set of add-ons that is used for each of the tests. That's fairly slow, so that's why it's the same for each, it's only created once for the entire test class.

Ultimately it's expected that changes to scoring/search algorithm will cause updates to multiple tests, that's fine. There is a script provided in this file to do bulk changes when required (In fact I expected to need to do more changes here, but ultimately didn't have to because of how much an edge case it is)